cdws_api/views.py

```diff
@@ -360,6 +360,30 @@ def calculate_counts(self, request, pk=None):
             data={'message': 'Calculation done.'},
             status=status.HTTP_200_OK)
 
+    @detail_route(methods=['post'],
+                  permission_classes=[IsAuthenticatedOrReadOnly])
+    def update_metrics(self, request, pk=None):
+        if 'metrics' in request.DATA and request.DATA['metrics'] != '':
+            if type(request.DATA['metrics']) is not dict:
+                return Response(
+                    status=status.HTTP_400_BAD_REQUEST,
+                    data={'message': 'Invalid metrics format'})
```

Comment:
Need to mode detailed response to user like: You send metrics - {request.DATA['metrics]}, but it should be object. etc